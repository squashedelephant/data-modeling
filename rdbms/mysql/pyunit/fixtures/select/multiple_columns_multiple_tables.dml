INSERT INTO transaction(name) VALUES('direct deposit');
INSERT INTO transaction(name) VALUES('atm deposit');
INSERT INTO transaction(name) VALUES('wire transfer');
INSERT INTO transaction(name) VALUES('atm withdrawal');
INSERT INTO transaction(name) VALUES('teller withdrawal');
INSERT INTO account(first_name, last_name, code) VALUES('John', 'Doe', 1111);
SET @acct_id = LAST_INSERT_ID();
SET @xact_id = (SELECT id FROM transaction WHERE name='atm deposit');
INSERT INTO account_history SET acct_id=@acct_id, xact_id=@xact_id, transaction_date=CURRENT_DATE(), transaction_time=CURRENT_TIME(), amount=100.00;
SET @xact_id = (SELECT id FROM transaction WHERE name='atm withdrawal');
INSERT INTO account_history SET acct_id=@acct_id, xact_id=@xact_id, transaction_date=CURRENT_DATE(), transaction_time=CURRENT_TIME(), amount=40.00;
