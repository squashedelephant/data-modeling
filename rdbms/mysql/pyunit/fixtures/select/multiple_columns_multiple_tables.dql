SELECT a.first_name, a.last_name, t.name, ah.amount FROM account a INNER JOIN account_history ah INNER JOIN transaction t ON t.id = ah.xact_id AND a.id = ah.acct_id AND ah.xact_id=2;
