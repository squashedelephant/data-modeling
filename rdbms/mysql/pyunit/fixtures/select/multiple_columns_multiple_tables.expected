John
Doe
atm deposit
100.00
